src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User[] users = UserService.getUsers();
```

Comment:
let's try to limit ourselves to using only the findByEmail method

Reply:
made new commit